ui/public/diagnoseReport.html

```diff
@@ -10,6 +10,8 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="%PUBLIC_URL%/supportedBrowsers.js"></script>
+    <script src="%PUBLIC_URL%/checkBrowser.js"></script>
```

Comment:
We can combine these 2 js together but that needs to use sed. As I know, the sed works differently between macOS and Linux.